feat(content): Saving Ildico - Human mission (#10033)
endless-sky/endless-sky: data/human/human missions.txt

			`	"Obviously, the details are proprietary, but in very broad generalities, he wanted me to warm up a freezing world."`
			`	Ildico doesn't blink; on second thought, you're not sure you've ever seen her blink. "Hope. Yes, and could you warm it?"`
			`	Ian laughs. "Oh, of course, a job like that would have been quite simple."`
			`	"Then why didn't you?" The tension is obvious in Ildico's voice.`
			`	Ian shrugs. "Though he could afford it there were other concerns."`
			`	Ildico steps forward and looms over Ian like an angry shadow. It appears that it is taking all of her willpower to keep her from ripping this man apart. "Concerns?"`
			`	"Y-yes. I don't know the specifics. You have to understand, here at the Academy it's not just about money. It's about politics, and power. I am just a Second Degree Terraformer. I wasn't privy to the details. But someone, or some government or corporation didn't want it. If you want to know more you'll have to ask him yourself."`
			`	Ildico seems to deflate. "Ask him? Is he here?"`

> 			`	The terraformer shakes his head. "Oh, no. He doesn't work for the Academy. But after the rejection he went to the heart of the Republic, I assume to find out why. He now works for them. I see his name from time to time. He, like all the real leaders of the Republic, is on Chiron. Just head to the main office for the Department of Migration."` # This is a rather huge precedent. Will need to either justify or restate.

Author: Fixed it.

			`	"Yes, boss."`
			`	"Got it, boss."`
			`	"Sure thing, Cap."`
			`	"Whatever you say, boss."`
			`	The group of pirates melt away into the warehouse, one of them dragging the dead body, leaving you and Ildy relatively alone.`
			`	"I'm so glad you came, <name>." Ildico puts a friendly hand on your shoulder and squeezes.`
			`	She turns and waves toward the warehouse. "This is my temporary headquarters. I have few other safe-houses, and own part of a hangar elsewhere. Not too many ships yet, but I intend to change that soon."`
			`	Ildico holsters her gun right next to where she keeps her dagger and continues. "But making moves is hard. Everyone wants to be on top. Which is why I need you."`
			`	She looks you in the eye and says, "There's a rival gang right next door to us. They have a lot more firepower, but if they could be removed, we could easily double our territory.`
			`	"I don't have the ships," she glances at her gang in the distance, "or the quality of crew, to deal with them."`
			`	"But, I think you do. I can't afford to pay you right now, things are pretty fluid, but I'll owe you. What do you say?"`
			choice
				`	"Sure, I'll help."`

> 				`	"Fine. But you're going to owe big when this is all over."`

Reviewer: \<first\> \<last\> still isn’t getting paid?

Author: Not everything is a payday ;)

			`	"I fed the gang some false information. They have their entire fleet right now looking for some juicy merchant ships that don't actually exist, so they're not expecting a captain like you to take them on and blow them out of the sky.`
			`	"Check out the nearby systems and look for a ship called the <npc>, that's their flagship. Kill them all, and come back here. To me."`
				accept

			label uninterested
			`	Ildico frowns. She looks down at her feet for a moment, and when she looks back up her face is a mask of rage and hurt. "Then get out."`
				decline

			label why?
			`	Acting defiantly in the face of so much danger, you reach for your sidearm. Unfortunately, it's in vain, because as soon as you feel the pistol grip in your hand you simultaneously feel the burning sensations of a flurry of bullets entering your body and tearing apart your insides.`
				die

	npc kill
		government "Bounty"
		personality heroic staying nemesis target
		system
			distance 1 2
		fleet "Marauder fleet IV"
		dialog phrase "generic hunted bounty fleet eliminated dialog"

	on visit
		dialog phrase "generic fleet bounty hunting on visit"

> 		dialog "You have destroyed the fleet. You can now return to <planet> to and report the news to Ildico."

Reviewer: ```suggestion
```

Author: Why the removal?

Reviewer: It is in `on visit`, so it can appear even if you have not, in fact, destroyed the fleet.

				decline

mission "Stone to Hope: Thanks"
	source "Norn"
	to offer
		has "Stone to Ice Queen: offered"
	on offer
		conversation
			`In the quiet, creaking spaceport, it's easy enough to spot Helm doing some much needed maintenance on the ships that have landed here for the night. The last time you met him, he asked you to bring a stone to Hope. Do you want to check in with him?`
			choice
				`	(Go over to Helm and tell about the stone.)`
					goto tell
				`	(Leave Helm alone.)`
					decline
			
			label tell
			`	As you say your hello you accidentally startle Helm, who's in the middle of recalibrating a repulsor.`
			`	"Woah! <first>! No need to sneak up on me!" He stands up to greet you. "What's new?"`
			choice
				`	"I found the Family-Cairn and left the stone there."`
				`	"I chucked the stone out of a hatch, but it's at least on Hope now."`
					to display
						has "chucked tosti's stone"
				`	"I threw the stone off a cliff, but it's at least on Hope now."`
					to display
						has "threw tosti's stone"
				`	"Where's Tosti?"`

> 			`	"Ah, the lad's out on the seas. Got himself a job working on a fishing trawler. It'd likely be a few years before he'll call to port here again."`

Reviewer: I don’t think this follows unless you choose the fourth dialogue option.

Author: Alright. I've added an additional outcome if you were a jerk.

			`	She starts off to join River and Bunker out in the spaceport before turning around one last time in the doorway. "We might be in need of you for some future projects too, Captain <last>. But goodbye for now. Stay safe on the star lanes!"`

event "stone waiting most"

mission "Stone to Hope"
	name "Bring Tosti's stone to <planet>"
	description "Take Tosti's family stone to <destination>, where they used to live before they immigrated to <origin>."
	to offer
		random < 60
		has "Stone of our Fathers 5: done"
		has "event: stone waiting most"
		not "Ice Queen: offered"
	source "Norn"
	destination "Hope"
	on offer
		conversation
			`There is a certain comfort to be found in the familiar swaying of Norn's spaceport. The creaking bridges no longer feel intimidating or threatening, only welcoming. There's never much staff here, but the few you do run into greet you warmly. Word has apparently spread that you've all but been adopted into the Hlewagasti family.`

> 			`	Helm, with his sandy hair and short beard, approaches you with a young man in tow. "<first>! It's well to see you again.`

Author: I don't think this is incorrect, just not as popular. But I like the idea of Norn's residents sounding a little different.

			`	"So..." he says conspiratorially, "... there is no date yet, but a certain red haired stoic and his exotic Alfheimian girlfriend may be looking at wedding dates."`
			choice
				`	"That's great news!"`
				`	"Do you know when?!"`
				`	"Who are you talking about?"`
					goto who
				`	"I don't really care."`
					goto cold

			`	"Like I said, there is no date yet, and with those two it could be years before they finally land on a date. Time's different out on the algae islands." He shrugs.`
			`	"But I thought you'd appreciate a small update on Thorleif." He smiles and then glances at the young man next to him. "Anyways, that's not actually why I sought you out."`
				goto hopestone

			label who
			`	"You don't remember Thorleif? My cousin out on the algae islands?" He frowns, but regains his composure and glances at the young man next to him. "Anyways, that's not actually why I sought you out."`
				goto hopestone

			label cold
			`	Helm frowns and takes a step back like he's been struck. After a moment he regains his composure and replies, "If you say so. In that case I have another matter to discuss with you."`

			label hopestone

> 			`	Standing beside Helm is a young, unremarkable man with a sunburnt face, calloused hands, and threadbare garb. "Ca-Captain <last>?" he says nervously, "I'm sorry to be a bothering you, ser. But I heard tell what you did for Thorleif, and I thought maybe you were the like to do a good turn by me."`

Author: you sure there should be two 'ands' in this phrase?

			`	Sensing his discomfort Helm puts an arm on the young man's shoulder. "It's alright, Tosti, we're all friends on Norn."`

> 			`	The young man staightens up a bit at Helm's reassurance and looks you in the eyes. "My da, well, he liked the waves well enough, but never really settled here, you see? In his heart that is.`

Author: It's da', his da'.

			`	Tosti is red and shaking and mumbles some apologies before slinking away from you and Helm.`
			`	Helm gives you a nod and says, "Thanks for listening, at least."`
				decline

			label hopeprice
			`	He grimaces, as if he was just punched in the gut. "I am sorry, ser. We're just poor laborers. Work is hard to come by for us refugees. I thought - I thought I'd just ask. In case you were near. In case it was within your power is all."`
			`	His eyes remain downcast and tears slide down his smooth face. His whole body seems to shake with the shame of what he just admitted.`
			choice
				`	"I might be in the area. No promises though.`
				`	"Alright. I'll do it."`
				`	"Sorry. I don't work for free."`
					goto hopeno

			label hopeyes
			`	The young man's eyes light up as he drags them off the floor and focuses on your face once more. "Really? Truly? I didn't dare hope..."`
			`	His body weight shifts and it looks as though he's about to hug you before he abruptly stops himself and tries to regain a little composure.`

> 			`	"Thorleif was right about you," he says as he presses the small, cool stone into your hand. "The Family-Cairn is not far from where the last spaceport used to be. Just by the cliffs, family name was Snaiwaz."`

Author: This is meant to just be him talking. He's not speaking in correct full sentences. He's just saying ideas as they come to him.

	`	Ildico sighs. "A treasure hunter killed my brother a few years back. My mother... she is also dead, and I am starving. I have no way to contact anyone else, and no one else is left."`
	`	"So, what do you say." She leans her head back, clearly exhausted by this exchange, and closes her eyes. "Will you kill me? Take advantage of me? Save me? Or leave me... to die."`
	choice
		`	"Come with me. It's time you gave up on Hope."`
			goto comewithme
		`	"I'll get you out of here. After that, no promises."`
			goto comewithme
		`	"I don't have time for wretches. You're on your own."`
		`	"How about I kill you, and take your stuff.`

	`	Ildico, her eyes still closed, makes a swift motion with her hand and flicks her wrist toward you. Before you have a chance to react, her knife is embedded deep in your throat.`
	`	You're on the ground now. Ruby red blood oozing out over the cold floor. You hear a strange moan escaping from your mouth, and Ildico's hands are in your pockets removing everything you have of value.`

> 	`	As the world fades to black, you hear her talking, though her words sound as though they are coming from a great distance. "Like I said, I won't die here. Thanks for the ship."`

Author: That's her problem, not yours. lol

	`	She gives you a stern nod, and then exits your ship and into the chaos of Clark.`
		decline

	label helpildico
	`Your thrusters kick up fresh dust as you land on your assigned concrete pad on the fringes of the sprawling spaceport. Ildico, looking less pale, and significantly more sturdy, waits by the exit.`
	`	"The terraforming firm is certain to have an office, or booth, in the ramshackle spaceport here. Let's pay them a visit."`

mission "Ice Queen 2"
	name "Find Ildico's Father"
	description "Look for Ildico's father on <destination>, where he went after he left Clark."
	source "Clark"
	destination "Glory"
	passengers 1
	to offer
		has "helpildico"
		or
			has "Ice Queen: done"
			has "Stone to Ice Queen: done"

	on offer
		conversation
			`Pushing through the boomtown hustle and bustle, you and Ildico make your way to a series of kiosks, booths, and hastily constructed offices that serve a variety of needs and functions in the ever-growing world of Clark.`
			`	Among these you find the offices of "Clark's Third Law Terraformists." Ildico steps in front of you and enters first.`

> 			`	An older man with greying hair and a bit of a paunch looks up from his desk. He stands up genially as both of you approach and says, "Howdy friends, I am Eustice Goodfellow, and welcome to our cozy establishment. Do you have need of a little terraforming?" Eustice flashes a wide open smile.`

Author: Okay, changed it.

	to offer
		has "Ice Queen 2: done"
	source "Glory"
	destination "Chiron"
	passengers 1
	on offer
		conversation
			`The Academy of Planetary Sciences is a campus the size of a small nation-state on Earth. Various artificially created and maintained microclimates abound as you travel between miniature mountains and micro-savannahs; between marshland and jungle, desert and forest. Impossible machines - some larger than skyscrapers - litter the twisted and ever-changing landscape, and terraforming satellites make meteor showers overhead in the dawn light. This is a place where anything is possible, and land bends eagerly to the whims of the terraforming wizards.`
			`	Eventually, Ildico and you make it to the primary reception complex, where after hours of filling out inane forms, and waiting in different meaningless queues, you finally reach someone who can help you. An old visitor list shows that one Deoric Drifa had visited a Professor Ian McGiven, Second Degree Terraformer and World Warming specialist in JZ86 Cantos-B of Cooling Block 9 in the Everest Facility.`

> 			`	The receptionist helpfully draws you a map to where you're going. She also lets you know that in order to visit him you'll need to fill out six other forums first. You get to work.`

Author: Mutli-world Bureaucracy is Bureaucracy squared.

mission "Ice Queen 4"
	landing
	invisible
	source "Chiron"
	passengers 1
	to offer
		has "Ice Queen 2: done"
	on offer
		event "deoricdeeds" 1103
		conversation
			`It's easy to imagine Earth once look and felt much like Chiron does today. A vibrant world, full of engaged and motivated people, all in a hurry to change the universe, or at least, to run it.`
			`	The Department of Migration, Stellar Populations and Advanced Interplanetary Economics is easy enough to find. What's surprising is how quickly you are ushered into a waiting room outside the office of one Mr. Drifa, Sub-Executive of Multiplanetary Pricing.`
			`	After a brief wait, a young secretary announces that Mr. Drifa is ready to see you, and brings you and Ildico into his office.`
			`	"So what can I help you with..." Deoric Drifa's voice trails off as he sees Ildico.`
			`	Ildico tries to step forward than stops herself. She freezes up.`
			`	"It can't be," the elder Drifa whispers.`
			`	"Why. Didn't. You. Come?" Ildico forces out each word.`
			`	"I sent for you. They told me you were all dead. They found bodies... and the generator blown."`

> 			`	"The accident..." Ildico says more to herself than anyone else. And then in that instant she unfreezes, emotions roil through her, and she runs to her father. Everything forgiven.`

Author: I think it works better having it be a separate phrase.. but if someone else wants to weigh in on this I'd appreciate it.

> 			`	"The accident..." Ildico says more to herself than anyone else. And then in that instant she unfreezes, emotions roil through her, and she runs to her father. Everything forgiven.`

Reviewer 2: I'd lean more to Giraffes suggestion.

mission "Ice Queen 6"
	name "Take Ildico to <planet>"
	description "Smuggle Ildico and her child to <destination>."
	source "Chiron"
	destination "Haven"
	passengers 2
	illegal 1250000 `Ildico Drifa is dragged away in chains for high crimes against the Republic and its most honorable servants. Her infant child is taken into the custody of the Republic. You can only watch in sad silence as they are removed from your ship. You wind up quite a bit poorer through a combination of fines and bribes (the breakdown remains somewhat unclear).`
	stealth
	to offer
		has "Ice Queen 5: done"

	on offer
		conversation
			`The directions Ildico gave you take you beyond the bustling city and out into the the large estates owned by the Republic political elite; Drifa Manor sits atop a low hill with what must be a very expensive view of New Sydney. The architecture of the building is reminiscent of elite villas you've seen in the Deep.`

> 			`	As you approach the main gate you find it wide open, and the attached security booth is abandoned. You could just walk right in.`

Author: I think could is better here.. otherwise it sounds like we are overtly telling the player what to do.

> 			`	As you approach the main gate you find it wide open, and the attached security booth is abandoned. You could just walk right in.`

Author: It's a leading observation, its why open and you, or anyone, could just walk right in...

			choice
				`	"Thank you."`
					goto "gotta go"
				`	"Is it wise for you and your child to be left alone in a place like Haven?"`

			`	She laughs. "After surviving on Hope and Chiron? Haven will be a cakewalk."`
				goto end

			label "gotta go"
			`	"No, it is my appreciation that has to be given. You have done what few would have the conviction to do."`

			label end
			`	She takes her child in one hand, and the briefcase in the other, and walks off your ship.`

event "pirateildy"

mission "Ice Queen 7"
	landing
	name "Meet Ildy on <planet>"
	description "See what Ildico needs on <destination>."
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	destination "Haven"
	to offer
		has "event: pirateildy"
	on offer
		dialog
			`Your inbox blinks with a new message from Ildico from Haven:`
			``
			`	"It's been too long, and I could use your help. Please, come to <planet>.`
			`				-your friend`
			`					Ildy"`

	on complete
		dialog
			`Even from orbit, Haven looks ugly, and as you approach the planet it doesn't get any better. Upon landing you're alerted Ildico is waiting for you in the spaceport.`

mission "Ice Queen 8"

> 	name "Defeat Ildico's rival fleet"

Author: This was something Bene added. I don't really know the difference, or why one is used over the other, and am happy to make whatever change is best.

> 	name "Defeat Ildico's rival fleet"

Author: I guess in that case I'd just leave it the way Bene wrote it.

> 	name "Defeat Ildico's rival fleet"

Reviewer 3: I don't mind this change, though I'd leave rivals uncapitalised.
```suggestion
	name "Defeat Ildico's rivals"
```

			`	Professor Grenwhich Millstone, foremost authority on the Economy and History of the Northern Pariah worlds, had this to say about Drifa:`
			`	"It is patently inconceivable that any single magnate could consolidate power on Haven. It is culturally anathema, and economically impractical. Frankly, all this talk of a so-called 'pirate queen' is just sensationalist journalism trying to sell the news. In a few weeks another pirate captain will be the new boogieman, and then another."`
			`	The segment ends, and news kiosk automatically scrolls down to the next article.`
				decline

mission "Skadenga: Blood of Surtr"
	landing
	source "Windblain"
	to offer
		has "Ice Queen 6: done"
		has "outfit: Blood of Surtr"
	on offer
		conversation
			`As your ship descends through Windblain's cloudy atmosphere, you consider landing near Hrithfjall and asking the Ondurdis about the Blood of Surtr.`
			choice
				`	(Take the Blood of Surtr to Hrithfjall.)`
					goto hrithland
				`	(Ask about the Blood of Surtr another time.)`

> 				`	(Don't visit Hrithfjall.)`

Author: I feel like the at all may be redundant here.

> 				`	(Don't visit Hrithfjall.)`

Author: Alright. I'll make the change.

					decline

			`	You fly past Hjrithfjall without landing and continue on your way.`
				defer

			label hrithland
			`	The pass numerous new homes built of driftwood and sea-stone as you approach the old village center of Hrithfjall. There is laughter and song in the air and children running everywhere; it always seems larger every time you visit it.`
			`	At the old village hall you find the Ondurdis and Aldfrith in conversation with a band of people who quite stick out here, foreigners. The Ondurdis's old eyes light up and she gives you a grandmotherly smile as you approach.`
			`	"<first>!" the Ondurdis exclaims happily. "My heart sings to see you." `
			choice
				`	"Who are they?"`
					goto who
				`	"I've come to show you something."`
					goto something
				`	"What do you know about other stones like the Tear of Skade?"`
			`	"On Skade? Only Tears of Skade. But on other worlds? Maybe.`
				goto something

			label who
			`	The Ondurdis steps away from the group and takes you aside, leaving Aldfrith to continue talking with them.`

> 			`	"They from Alfheim. They say their culture and world dying and they look for new home before old one gone." She pats your arm gently.`

Author: What's missing?

> 			`	"They from Alfheim. They say their culture and world dying and they look for new home before old one gone." She pats your arm gently.`

Author: It was.. but.. I forget her speech patterns sometimes, lol. She does use 'are', so I'll add it back in.

			choice
				`	"What happened to them?"`
				`	"Where could I find them?"`

			`	Aldfrith shakes his head slowly. "In the time of my grandfather the men from the Deep came and took them off world. They said they were too valuable to belong in the hands of savages, and since then we've never found another Scale of Ran. I think she was displeased we allowed them to be taken."`
			`	The old man pauses for a moment and then says, "Although.. I do remember when I was a child the folk from the Deep coming and visiting us again. They had questions about.. our baubles. I think they said they did not work? They thought they were just useless charms. Perhaps the Scales stopped working?"`
			`	He shrugs. "I am sorry, <first>, that I cannot tell you more."`
			`	The Ondurdis puts a hand on his back. "I value your stories." She turns her attention back to you, "Maybe all God-Stones once produced wondrous stones of power."`
			`	She points at the Blood of Surtr. "Maybe that is where yours came from. But more than guesses, we cannot provide."`
			choice
				`	(Take your leave.)`

> 			`	You thank the Ondurdis and Aldfrith for their gracious help and interesting stories. The Ondurdis hastily makes you a meal to take with you, believing it bad manners to let you leave unfed and wishes you well in your future travels among the lofty stars.`

Author: I changed it, does it feel better now?

> 			`	You thank the Ondurdis and Aldfrith for their gracious help and interesting stories. The Ondurdis hastily makes you a meal to take with you, believing it bad manners to let you leave unfed and wishes you well in your future travels among the lofty stars.`

Reviewer 4: Much.

			choice
				`	(Follow her in.)`

			label ridealong
			`	After a brief ride through the campus you are escorted deeper and deeper into the maze that is the Everest Facility. Another half an hour of hallways, and turns, and stairs and ups and downs, you finally find yourself inside Ian McGiven's office.`
			`	The man hears your inquiry and nods earnestly. "Ah, yes, of course I remember the fellow! What an interesting meeting that was," Ian says from behind his small desk.`
			`	"Obviously, the details are confidential, but in very broad generalities, he wanted me to warm up a freezing world."`
			`	Ildico doesn't blink; on second thought, you're not sure you've ever seen her blink. "Hope. Yes, and could you warm it?"`
			`	Ian laughs. "Oh, of course, a job like that would have been quite simple."`
			`	"Then why didn't you?" The tension is obvious in Ildico's voice.`
			`	Ian shrugs. "Though he could afford it there were other concerns."`
			`	Ildico steps forward and looms over Ian like an angry shadow. It appears that it is taking all of her willpower to keep her from ripping this man apart. "Concerns?"`

> 			`	"Y-yes. I don't know the specifics. You have to understand, here at the Academy it's not just about money. It's about politics, and power. I am just a Second Degree Terraformerm, and after the initial meeting I wasn't privy to the details. But someone, or some government or corporation didn't want it. If you want to know more you'll have to ask him yourself."`

Reviewer 4: ```suggestion
			`	"Y-yes. I don't know the specifics. You have to understand, here at the Academy it's not just about money. It's about politics, and power. I am just a Second Degree Terraformer, and after the initial meeting I wasn't privy to the details. But someone, or some government or corporation didn't want it. If you want to know more you'll have to ask him yourself."`
```

Author: thanks, fixed it

			choice
				`	"Ildico, what's going on?"`
				`	"Are you hurt? Do you need help?"`
				`	"What have you done?"`
				`	(Say nothing.)`

			`	She ignores you, her attention focused elsewhere.`
			`	"You thought," Ildico says as she catches her breath, "that you could sell me? That you could give me to the bureaucrat who signed Hope's death warrant? That I would be your pawn, your tool, your object in order to gain favor and promotion?"`
			`	The raven-haired woman laughs, as she cleans off her bloody dagger, and takes a step toward a figure cowering in the corner. The man you recognize as her father.`
			`	"Wh-what have you done? You've ruined everything. I was so close. I would have had it all." Deoric blinks, his face sets in hard, and he straightens up, he looks as if he just remembered who he is.`
			`	"You stupid, selfish, arrogant brat. You just needed to do as you were told. To play your pathetic part. You couldn't could you? You were always worthless. If only your brother had survived." Deoric picks up a broken furniture leg and confidently wields it like a club.`

> 			`	At this Ildico's wicked grin widens deeper. "My brother? Dear Hogne. Father, I killed him. He couldn't handle the ice, the isolation, the hopelessness of Hope. He lost his mind. He tried to make me his wife. And mother? She cracked long before that. An entiteled debutant like her had no chance in that hellhole, and I? I needed to eat."`

Reviewer 4: ```suggestion
			`	At this Ildico's wicked grin widens deeper. "My brother? Dear Hogne. Father, I killed him. He couldn't handle the ice, the isolation, the hopelessness of Hope. He lost his mind. He tried to make me his wife. And mother? She cracked long before that. An entitled debutant like her had no chance in that hellhole, and I? I needed to eat."`
```

			`	The color drains out of Deoric's face. "Monster." He says. "I will put an end to you. I will start a new family, as I should have before."`
			`	Deoric lunges at Ildico, but before he can take more then a single step, Ildico's wrist flicks and her knife slams into Deoric's throat. He falls to the ground, bleeding out, dying.`

> 			`	Ildico stands over him. "You are a coward. A miserable disgusting man." She spits on him, and pulls her dagger from his heaving neck. He tries to speak as his dying body squirms, but no words materilize and he goes still."`

Reviewer 4: ```suggestion
			`	Ildico stands over him. "You are a coward. A miserable disgusting man." She spits on him, and pulls her dagger from his heaving neck. He tries to speak as his dying body squirms, but no words materialize and he goes still."`
```

			`	Ildico steps forward and looms over Ian like an angry shadow. It appears that it is taking all of her willpower to keep her from ripping this man apart. "Concerns?"`
			`	"Y-yes. I don't know the specifics. You have to understand, here at the Academy it's not just about money. It's about politics, and power. I am just a Second Degree Terraformer, and after the initial meeting I wasn't privy to the details. But someone, or some government or corporation didn't want it. If you want to know more you'll have to ask him yourself."`
			`	Ildico seems to deflate. "Ask him? Is he here?"`
			`	The terraformer shakes his head. "Oh, no. He doesn't work for the Academy. But after the rejection he went to the heart of the Republic, I assume to find out why. He now works for them. I see his name from time to time. Just head to the the Republic's main office for the Department of Migration which is located on Chiron."`
			`	Without saying another word, Ildico turns and leaves. You thank Ian for his help and follow her out.`

mission "Ice Queen 4"
	landing
	invisible
	source "Chiron"
	passengers 1
	to offer
		has "Ice Queen 2: done"
	on offer
		event "deoricdeeds" 1103
		conversation

> 			`It's easy to imagine Earth once looked and felt much like Chiron does today. A vibrant world, full of engaged and motivated people, all in a hurry to change the universe, or at least, to run it.`

Reviewer 3: ```suggestion
			`It's easy to imagine Earth once looked and felt much like Chiron does today. A vibrant world, full of engaged and motivated people, all in a hurry to change the universe; or at least, to run it.`
```

				decline

			label why?
			`	Acting defiantly in the face of so much danger, you reach for your sidearm. Unfortunately, it's in vain, because as soon as you feel the pistol grip in your hand you simultaneously feel the burning sensations of a flurry of bullets entering your body and tearing apart your insides.`
				die

	npc kill
		government "Bounty"
		personality heroic staying nemesis target
		system
			distance 1 2
		fleet "Marauder fleet IV"
		dialog phrase "generic hunted bounty fleet eliminated dialog"

	on visit
		dialog phrase "generic fleet bounty hunting on visit"

	on complete
		conversation
			`There's been some intense fighting around the district where Ildico's warehouse was located. A few gunshots echo as you make your way through the rubble and around the fire.`
			`	A group of injured, but high-spirited pirates are set up outside the building, and there are bodies littering the nearby streets. An unharmed but bloody Ilidco walks up to you, smiling. She is trailed by a small incongruous toddler with short black hair and lopsided grin.`
			`	"Perfect. They never had a chance." She wipes the blood off her dagger, and notices your gaze.`

> 			`	"The safest place for her was by my side." She says matter of factly, "Randalin this is <first>, you met them long ago when you were a baby. <First> is my friend."`

Reviewer 3: ```suggestion
			`	"The safest place for her was by my side," she says matter-of-factly to you, then she kneels down to her daughter. "Randalin, this is <first>. You met them long ago when you were a baby. <first> is my friend."`
```

> 			`	She turns back to you, "And <first>, this is Randalin. She's grown a bit since you saw her last."`

Reviewer 3: ```suggestion
			`	She stands up again and looks back at you. "And <first>, this is Randalin. She's grown a bit since you saw her last."`
```

			choice
				`	"Hi, Randalin. It's wonderful to meet you."`
					goto hirand
				`	(Nod at the child.)`
					goto onlyyou
				`	"She was with you for the battle?"`

> 			`	Ildico shrugs. "Who would I trust with her? Who could I trust with her. Only you, <first>, and you were elsewhere."`

Reviewer 3: ```suggestion
			`	Ildico shrugs. "Who would I trust with her? Who could I trust with her? Only you, <first>, and you were elsewhere."`
```

				goto onlyyou

			label hirand
			`	The little girl lifts a chubby fist up to you.`
			choice
				`	(Fist bump her.)`
					goto bump
				`	(Shake her fist.)`
			`	You awkwardly take her fist in your hand and shake it up and down. Randalin giggles, and babbles something incoherent.`
				goto words
			
			label bump
			`	Her small fist bumps against yours and then she gives you an easy grin and babbles something incoherent.`

			label words

> 			`	"She's not quite got the hang of words yet. But they'll come." Ildico says approvingly.`

Reviewer 3: ```suggestion
			`	"She's not quite got the hang of words yet. But they'll come," Ildico says approvingly.`
```

			`	The segment ends, and the news kiosk automatically scrolls down to the next article.`
				decline

mission "Skadenga: Blood of Surtr"
	landing
	source "Windblain"
	to offer
		has "Ice Queen 6: done"
		has "outfit: Blood of Surtr"
	on offer
		conversation
			`As your ship descends through Windblain's cloudy atmosphere, you consider landing near Hrithfjall and asking the Ondurdis about the Blood of Surtr.`
			choice
				`	(Take the Blood of Surtr to Hrithfjall.)`
					goto hrithland
				`	(Ask about the Blood of Surtr another time.)`
				`	(Don't visit Hrithfjall at all.)`
					decline

			`	You fly past Hjrithfjall without landing and continue on your way.`
				defer

			label hrithland
			`	You pass numerous new homes built of driftwood and sea-stone as you approach the old village center of Hrithfjall. There is laughter and song in the air and children running everywhere; it always seems larger every time you visit it.`

> 			`	At the old village hall you find the Ondurdis and Aldfrith in conversation with a band of people who quite stick out here, foreigners. The Ondurdis's old eyes light up and she gives you a grandmotherly smile as you approach.`

Reviewer 4: ```suggestion
			`	At the old village hall you find the Ondurdis and Aldfrith in conversation with a band of people who quite stick out here, seemingly foreigners. The Ondurdis's old eyes light up and she gives you a grandmotherly smile as you approach.`
```

Author: I agree that it needs to be spruced up, but I found seemingy a bit clunky. Check my recent commit and see if that is more to your likening.

> 			`	"<first>!" the Ondurdis exclaims happily. "My heart sings to see you." `

Reviewer 3: ```suggestion
			`	"<first>!" the Ondurdis exclaims happily. "My heart sings to see you."`
```

				decline

mission "Stone to Hope: Thanks"
	source "Norn"
	to offer
		has "Stone to Ice Queen: offered"
	on offer
		conversation
			`In the quiet, creaking spaceport, it's easy enough to spot Helm doing some much needed maintenance on the ships that have landed here for the night. The last time you met him, he asked you to bring a stone to Hope. Do you want to check in with him?`
			choice
				`	(Go over to Helm and tell about the stone.)`
					goto tell
				`	(Leave Helm alone.)`
					decline
			
			label tell
			`	As you say your hello you accidentally startle Helm, who's in the middle of recalibrating a repulsor.`
			`	"Woah! <first>! No need to sneak up on me!" He stands up to greet you. "What's new?"`
			choice
				`	"I found the Family-Cairn and left the stone there."`
					goto where
				`	"I chucked the stone out of a hatch, but it's at least on Hope now."`
					goto threw
					to display
						has "chucked tosti's stone"
				`	"I threw the stone off a cliff, but it's at least on Hope now."`
					goto threw
					to display
						has "threw tosti's stone"
				`	"Where's Tosti?"`
			label where

> 			`	"Ah, the lad's out on the seas. Got himself a job working on a fishing trawler. It'd likely be a few years before he'll call to port here again."`

Reviewer: ```suggestion
			`	"Tosti's out on the seas. Got himself a job working on a fishing trawler. It'd likely be a few years before he'll call to port here again."`
```
This works regardless of if you ask where Tosti is

			`	Helm gives you a friendly smile. "Still, you've done him a good turn. I'll have a message sent to him so he knows he can rest easy."`

> 			`	He returns to wrestling with the rope but continues, "There's no money or reward, but word will spread and people here will appreciate it all the same. Here on Norn we have long memories."`

Reviewer 3: ```suggestion
			`	He returns to tinkering with the repulsor but continues, "There's no money or reward, but word will spread and people here will appreciate it all the same. Here on Norn we have long memories."`
```

				decline

			label threw
			`	Helm stops what's he doing and looks up at you searchingly. "Best keep that to yourself, then. You of all people should know that folks around here take stone-bearing pretty seriously."`
			`	He sighs, "Still, better than not being on Hope at all, I suppose. As for Tosti, he's out on the seas. Got himself a job working on a fishing trawler. It'd likely be a few years before he'll call to port here again."`
			`	He turns and looks out over Norn's endless oceans. "I'll have a message sent to him so he knows he can rest easy. I'll of course leave out the part about you tossing the stone. No reason to give him grief."`

> 			`	He returns to wrestling with the rope but continues, "There's no money or reward, but a good deed is it's own reward, isn't it?"`

Reviewer 3: ```suggestion
			`	He returns to tinkering with the repulsor but continues, "There's no money or reward, but a good deed is its own reward, isn't it?"`
```

			`Ale, River, and Bunker look glad to be back on <planet>. "Being there in person was bad enough," Ale grumbles as she looks through a stack of folders on her datapad. "Now we have to watch it over and over again to edit it into something presentable."`
			`	As they are leaving, Ale hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you. We really couldn't have done it without you," she tells you with a bittersweet smile. "Of course, you'll be one of the first to know when it's done."`
			`	She starts off to join River and Bunker out in the spaceport before turning around one last time in the doorway. "We might be in need of you for some future projects too, Captain <last>. But goodbye for now. Stay safe on the star lanes!"`

event "stone waiting most"

mission "Stone to Hope"
	name "Bring Tosti's stone to <planet>"
	description "Take Tosti's family stone to <destination>, where they used to live before they immigrated to <origin>."
	to offer

> 		random < 60
> 		has "Stone of our Fathers 5: done"
> 		has "event: stone waiting most"

Reviewer: ```suggestion
		random < 60
		has "Stone of our Fathers 5: done"
		has "event: stone waiting most"
		not "Ice Queen: offered"
```

			`	With the stone firmly in your fist, he takes his leave from you, saying "thank you" over and over again before he's finally gone.`
			`	Helm silently watches him go and then turns back to you, shaking his head. "You're not like any off-worlder I've ever known. Most would have laughed in that boy's face, let alone done him a good turn for free."`
			`	He smiles at you and says, "You really are one of us."`
				accept

mission "Stone to Ice Queen"
	landing
	name "Bring Ildico to <planet>"
	description "You found a woman named Ildico Drifa in an abandoned manor on <origin>. She has asked you to bring her to <destination> so she can escape the abandoned planet."
	source "Hope"
	destination "Clark"
	passengers 1
	to offer
		has "Stone to Hope: done"
	on offer
		conversation "Ice Queen"
	on complete
		conversation "Ice Queen End"

mission "Ice Queen"
	landing
	name "Bring Ildico to <planet>"
	description "You found a woman named Ildico Drifa in an abandoned manor on <origin>. She has asked you to bring her to <destination> so she can escape the abandoned planet."
	source "Hope"
	destination "Clark"
	passengers 1
	to offer
		has "Expedition to Hope 2: done"

> 		not "Stone to Ice Queen: offered"

Reviewer: ```suggestion
		not "Stone to Hope: done"
```

		random < 15
	on offer
		conversation "Ice Queen"
	on complete
		conversation "Ice Queen End"

conversation "Ice Queen"

> 	branch "has stone"

Reviewer: ```suggestion
	branch "has stone"
		has "Stone to Hope: done"
```

			`	The old man pauses for a moment and then says, "Although.. I do remember when I was a child the folk from the Deep coming and visiting us again. They had questions about.. our baubles. I think they said they did not work? They thought they were just useless charms. Perhaps the Scales stopped working?"`
			`	He shrugs. "I am sorry, <first>, that I cannot tell you more."`
			`	The Ondurdis puts a hand on his back. "I value your stories." She turns her attention back to you, "Maybe all God-Stones once produced wondrous stones of power."`
			`	She points at the Blood of Surtr. "Maybe that is where yours came from. But more than guesses, we cannot provide."`
			choice
				`	(Take your leave.)`

			`	You thank the Ondurdis and Aldfrith for their gracious help and interesting stories. The Ondurdis hastily makes you a meal to take with you, believing it bad manners to let you leave unfed and wishes you well in your future travels among the lofty stars.`
				decline

mission "Stone to Hope: Thanks"
	source "Norn"
	to offer

> 		has "Stone to Ice Queen: offered"

Reviewer: ```suggestion
		has "Stone to Hope: done"
```

Reviewer: Ignore this

			`	Ildico gives you a sly smile. "It took years, but the majority of the Drifa fortune has been smuggled into this storage device. They thought I was meek, malleable, and pitiful. Not a threat. They were fools."`
			`	She holds up the box. "And in this, well, this may take some explaining. Before we migrated to Hope, the Drifa were old nobility in The Deep. Legend says we ruled Valhalla before the first ships came. Whatever the truth, my ancestors had this in their possession as a sign of their royalty, this bauble."`
			`	Ildico opens the rune-carved lid of the box, revealing a faintly glowing red rock. "It is called the Blood of Surtr, and it was said to have been the symbol of our rulership, and more importantly, a source of power."`
			`	She shrugs. "I know it sounds far-fetched. But I had this with me on Hope, and I don't think I would have survived without it. When the generators went down, this kept the lights on.`
			`	"But, it's time with me is over. I want you, the last friend of House Drifa, and the only person whose shown me any kindness in many years, to have it.`

> 			scene "outfit/bloodsurtr"

Reviewer: ```suggestion
			scene "outfit/surtrblood"
```

	to offer
		has "Expedition to Hope 2: done"
		not "Stone to Hope: done"
		random < 15
	on offer
		conversation "Ice Queen"
	on complete
		conversation "Ice Queen End"

conversation "Ice Queen"
	branch "has stone"
		has "Stone to Hope: done"
	`There is nothing here. Only unceasing desolation. The slow churning of hateful ice that swallowed the dream of a world. Yet, here you are. You've landed along the equator, where the environment is not quite so hostile, and where patches of unconquered ice-free rock still reign.`
	`	Beyond your ship are the ruins of the last unfrozen city. Where the final refugees huddled together awaiting their saviors, the Republic Navy, and were whisked away to other more welcoming worlds.`
	choice
		`	(Explore the area.)`
			goto explore
		`	(Leave this world.)`
	`	There is nothing here anyone could want. You leave this abandoned world to its glaciers and ghosts.`
		decline

	label explore
	`	You break out the cold gear and dress yourself in warm layers with sturdy boots. Furtive frost cracks and breaks as your ship's door opens, and you step out into the cold air. This world's dim light reflects painfully off the ice and snow, momentarily blinding you.`

> 	`	Once outside in the sharp, crisp air it's easy to find the dismal ruins of this world's last spaceport. The trek is slow and clumsy in the unfamiliar gravity and granular snow."`

Reviewer 4: ```suggestion
	`	Once outside in the sharp, crisp air it's easy to find the dismal ruins of this world's last spaceport. The trek is slow and clumsy in the unfamiliar gravity and granular snow.`
```

	label "has stone"
	`There is nothing here. Only unceasing desolation. The slow churning of hateful ice that swallowed the dream of a world. Yet, here you are. You've landed along the equator, where the environment is not quite so hostile, and where patches of unconquered ice-free rock still reign.`
	`	Beyond your ship are the ruins of the last unfrozen city. Where the final refugees huddled together awaiting their saviors, the Republic Navy, and were whisked away to other more welcoming worlds.`
	choice
		`	(Search for the Snaiwaz Family-Cairn.)`
			goto cairnsearch
		`	(Toss the stone out your hatch and leave this world.)`

	action
		set "chucked tosti's stone"
	`	The stone of Tosti's father tumbles out of the ship and comes to rest in a heap of cluttered refuse. This is as close as it will get to its Family-Cairn, and far closer than Tosti could have gotten it on his own.`
		decline

	label cairnsearch
	`	You break out the cold gear and dress yourself in warm layers with sturdy boots. Furtive frost cracks and breaks as your ship's door opens, and you step out into the cold air. This world's dim light reflects painfully off the ice and snow, momentarily blinding you.`

> 	`	Once outside in the sharp, crisp air it's easy to find the dismal ruins of this world's last spaceport. The trek is slow and clumsy in the unfamiliar gravity and granular snow."`

Reviewer 4: ```suggestion
	`	Once outside in the sharp, crisp air it's easy to find the dismal ruins of this world's last spaceport. The trek is slow and clumsy in the unfamiliar gravity and granular snow.`
```

	`	You work around the silent spaceport, where the half-buried hangars are littered with the forgotten mementos of desperate refugees, and many of the buildings are lost beneath the encroaching ice. Eventually you find the cliffs that Tosti spoke of, and the many dozens of Family-Cairns piled along side it. Silent watchers of the churning glaciers.`
	`	You see a nearly toppled Family-Cairn with the name "Snaiwaz" at its base.`
	choice
		`	(Place Tosti's father's stone.)`
			goto placestone
		`	(Throw the stone off the side of the cliff.)`

	action
		set "threw tosti's stone"

> 	`	You hurl the stone far beyond the cliff side and down into glacial slopes beneath it. Never to find rest by his kindred-stones.`

Reviewer 4: ```suggestion
	`	You hurl the stone far beyond the cliff side and down into glacial slopes beneath it. It is never to find rest by its kindred-stones.`
```

Reviewer 4: Yep, that works.

Author: done.

		decline

> 	label placestone

Reviewer 4: ```suggestion
	label placestone
	action
		set "placed tosti's stone"
```

	`	Eventually you find your way to the kitchen, which appears to be the epicenter of activity in this household. You see assorted survival rations and gear strewn about everywhere. More alarmingly, you also see a young woman sitting on the floor with her back against a counter.`
	choice
		`	"Who are you?"`
		`	"Are you okay?"`
		`	"What are you doing here?"`

	`	The woman has short jet black hair. She's wearing a military grade, all-purpose survival suit, the kind you might see on deeply embedded special forces. The only weapon on her is a decorated knife at her belt. She looks pale, and frail, probably malnourished. But her icy eyes are clear, and she locks them onto you.`
	`	"You come into my home, and expect me to answer your questions?" Her voice is steady.`
	`	Slowly, awkwardly, she pulls herself to her feet, keeping one hand on the counter for balance. She rests her other hand over her knife.`

> 	`	"You are lucky that you did not barge in here a year ago, or even a month ago." Despite her bravado there is a deep wariness behind her words. "Now tell me, who are you, and what are you doing here?" she says, through gritted teeth.`

Reviewer 4: ```suggestion
	`	"You are lucky that you did not barge in here a year ago, or even a month ago." Despite her bravado there is a deep wariness behind her words. "Now tell me, who are you, and what are you doing here?" she asks, through gritted teeth.`
```
Your choice on this.

		`	"My reasons are my own."`

	action
		set "helpildico"
	`	Ildico doesn't react for a moment, seemingly taking in your words.`
	`	At length she nods slowly. "We'll see."`
	`	She pauses again and then shrugs. "It's not like I have anything to lose... very well. I need to discover what happened to my father. I know he first went to Clark to meet with a terraforming firm there. That's where I start my search."`
		launch

conversation "Ice Queen End"
	branch helpildico
		has "helpildico"
	`Your thrusters kick up fresh dust as you land on your assigned concrete pad in the sprawling spaceport. Ildico, looking less pale and significantly more sturdy, waits by the exit.`
	`	"Thank you for getting me out of there. After all those years on the ice this place will be no challenge for me. Perhaps someday our paths will cross again. If they do, I will remember this small kindness."`
	`	She gives you a stern nod, and then exits your ship and into the chaos of Clark.`
		decline

	label helpildico
	`Your thrusters kick up fresh dust as you land on your assigned concrete pad on the fringes of the sprawling spaceport. Ildico, looking less pale, and significantly more sturdy, waits by the exit.`

> 	`	"The terraforming firm is certain to have an office, or booth, in the ramshackle spaceport here. Let's pay them a visit."`

Reviewer 4: ```suggestion
	`	"The terraforming firm is certain to have an office, or booth, in the ramshackle spaceport here. Let's pay them a visit."`
	label end
```

mission "Ice Queen 2"
	name "Find Ildico's Father"
	description "Look for Ildico's father on <destination>, where he went after he left Clark."
	source "Clark"
	destination "Glory"
	passengers 1
	to offer
		has "helpildico"
		or
			has "Ice Queen: done"
			has "Stone to Ice Queen: done"

	on offer
		conversation
			`Pushing through the boomtown hustle and bustle, you and Ildico make your way to a series of kiosks, booths, and hastily constructed offices that serve a variety of needs and functions in the ever-growing world of Clark.`
			`	Among these you find the offices of "Clark's Third Law Terraformists." Ildico steps in front of you and enters first.`
			`	An older man with greying hair and a bit of a paunch looks up from his desk. He stands up genially as both of you approach and says, "Howdy friends, I am Eustace Goodfellow, and welcome to our cozy establishment. Do you have need of a little terraforming?" Eustace flashes a wide open smile.`
			`	"Yes. I'd like to terraform the planet Hope." Ildico says flatly.`

> 			`	"Well, uh, ma'am. We don't, that is to say, we're not that kind of terraforming firm. We do small jobs. Touch ups. We can't.. we can't terraform a world, and I don't..." Eustace looks at both of you carefully before saying, "I don't think you could pay for it either."`

Reviewer 4: ```suggestion
			`	"Well, uh, ma'am. We don't, that is to say, we're not that kind of terraforming firm. We do small jobs. Touch ups. We can't... we can't terraform a world, and I don't..." Eustace looks at both of you carefully before saying, "I don't think you could pay for it either."`
```

			`	Eventually, Ildico and you make it to the primary reception complex, where after hours of filling out inane forms, and waiting in different meaningless queues, you finally reach someone who can help you. An old visitor list shows that one Deoric Drifa had visited a Professor Ian McGiven, Second Degree Terraformer and World Warming specialist in JZ86 Cantos-B of Cooling Block 9 in the Everest Facility.`
			`	The receptionist helpfully draws you a map to where you're going. She also lets you know that in order to visit him you'll need to fill out six other forms first. You get to work.`
			choice
				`	(Fill out the forms.)`
			choice
				`	(Fill out more forms.)`
			choice
				`	(Continue filling out forms.)`
			

> 			`The receptionist brings a few of the earlier forms you filled out and helpfully points out a few mistakes for you to fix.`

Reviewer 4: ```suggestion
			`	The receptionist brings a few of the earlier forms you filled out and helpfully points out a few mistakes for you to fix.`
```

			choice
				`	(Fix the forms you already filled out.)`
			choice
				`	(Fill out the final set of forms.)`

			`	A few painfully tedious hours later, all the right forms have been filled out and you're met by an intern in a small vehicle.`

> 			`	She glances at her electronic clipboard. "Captain <last>? Lady Drifa? I am here to escort you to the Everest Facility." She opens the door and gestures to you two. "Please climb aboard. I know the confines are quite small, but the E.F. is only a few kilometers away."`

Reviewer 4: ```suggestion
			`	She glances at her electronic clipboard. "Captain <last>? Lady Drifa? I am here to escort you to the Everest Facility." She opens the door and gestures to you two. "Please climb aboard. I know the confines are quite small, but the E.F. is a few kilometers away."`
```

			`	"No, <first>. I will protect you." Her voice is deadly serious. She then inspects the inside of the vehicle carefully and takes a seat. "It seems safe. And I need answers."`
			choice
				`	(Follow her in.)`

			label ridealong
			`	After a brief ride through the campus you are escorted deeper and deeper into the maze that is the Everest Facility. Another half an hour of hallways, and turns, and stairs and ups and downs, you finally find yourself inside Ian McGiven's office.`
			`	The man hears your inquiry and nods earnestly. "Ah, yes, of course I remember the fellow! What an interesting meeting that was," Ian says from behind his small desk.`
			`	"Obviously, the details are confidential, but in very broad generalities, he wanted me to warm up a freezing world."`
			`	Ildico doesn't blink; on second thought, you're not sure you've ever seen her blink. "Hope. Yes, and could you warm it?"`
			`	Ian laughs. "Oh, of course, a job like that would have been quite simple."`
			`	"Then why didn't you?" The tension is obvious in Ildico's voice.`

> 			`	Ian shrugs. "Though he could afford it there were other concerns."`

Reviewer 4: ```suggestion
			`	Ian shrugs. "Though he could afford it, there were other concerns."`
```

			`It's easy to imagine Earth once looked and felt much like Chiron does today. A vibrant world, full of engaged and motivated people, all in a hurry to change the universe; or at least, to run it.`
			`	The Department of Migration, Stellar Populations and Advanced Interplanetary Economics is easy enough to find. What's surprising is how quickly you are ushered into a waiting room outside the office of one Mr. Drifa, Sub-Executive of Multiplanetary Pricing.`
			choice
				`	(Wait for Mr. Drifa.)`

			`	After a brief wait, a young secretary announces that Mr. Drifa is ready to see you, and brings you and Ildico into his office.`
			`	"So what can I help you with..." Deoric Drifa's voice trails off as he sees Ildico.`
			`	Ildico tries to step forward than stops herself. She freezes up.`
			`	"It can't be," the elder Drifa whispers.`
			`	"Why. Didn't. You. Come?" Ildico forces out each word.`
			`	"I sent for you. They told me you were all dead. They found bodies... and the generator blown."`
			`	"The accident..." Ildico says more to herself than anyone else. And then in that instant she unfreezes, emotions roil through her, and she runs to her father. Everything forgiven.`

> 			`	After a long tear-filled embrace she disengages and wipes her eyes. "Early on the generator blew and we had to evacuate the compound. Some people died.. including Hogne." She glances at you and says, "He was my older brother."`

Reviewer 4: ```suggestion
			`	After a long tear-filled embrace she disengages and wipes her eyes. "Early on the generator blew and we had to evacuate the compound. Some people died... including Hogne." She glances at you and says, "He was my older brother."`
```

> 			`	She continues, "I salvaged what I could and set up at home. I've been at home the whole time."`

Reviewer 4: ```suggestion
			`	She continues, "I salvaged what I could and set up at home. I've been there the whole time."`
```

				`	"You're welcome, Ildico."`

			`	She looks at you for just a moment, as if in thought, and then says, "If we ever meet again, call me Ildy. It's what my friends call me."`
			`	With that you take your leave.`
				decline

event "deoricdeeds"

mission "Ice Queen 5"
	landing
	name "Visit Ildico"
	description "Ildico has asked you to visit her on <destination>."
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	destination "Chiron"
	to offer
		has "event: deoricdeeds"

	on offer
		dialog
			`Your inbox blinks with a new message from Ildico on Chiron:`
			``
			`	"I need to see you. Please come. My address is attached.`
			`				-your friend`
			`					Ildy"`

	on complete
		dialog
			`The myriad lights of urban settlements twinkle in the dusk as you approach the capital, and main spaceport, of Chiron. Crickets, brought from old Earth, chirp in the twilight air. Your ship parked, you'll need to make your way to the main spaceport to hire transportation to Ildico's address.`

mission "Ice Queen 6"
	name "Take Ildico to <planet>"
	description "Smuggle Ildico and her child to <destination>."
	source "Chiron"
	destination "Haven"
	passengers 2

> 	illegal 1250000 `Ildico Drifa is dragged away in chains for high crimes against the Republic and its most honorable servants. Her infant child is taken into the custody of the Republic. You can only watch in sad silence as they are removed from your ship. You wind up quite a bit poorer through a combination of fines and bribes (the breakdown remains somewhat unclear).`

Reviewer 4: This could run into problems where the Wanderers or Heliarch fine you for high treason against the Republic.
Probably not something that can be fixed at the moment, but just thought I'd point it out.

Author: :|

			`	"Wh-what have you done? You've ruined everything. I was so close. I would have had it all." Deoric blinks, his face sets in hard, and he straightens up, he looks as if he just remembered who he is.`
			`	"You stupid, selfish, arrogant brat. You just needed to do as you were told. To play your pathetic part. You couldn't could you? You were always worthless. If only your brother had survived." Deoric picks up a broken furniture leg and confidently wields it like a club.`
			`	At this Ildico's wicked grin widens deeper. "My brother? Dear Hogne. Father, I killed him. He couldn't handle the ice, the isolation, the hopelessness of Hope. He lost his mind. He tried to make me his wife. And mother? She cracked long before that. An entitled debutant like her had no chance in that hellhole, and I? I needed to eat."`
			`	The color drains out of Deoric's face. "Monster." He says. "I will put an end to you. I will start a new family, as I should have before."`
			`	Deoric lunges at Ildico, but before he can take more then a single step, Ildico's wrist flicks and her knife slams into Deoric's throat. He falls to the ground, bleeding out, dying.`

> 			`	Ildico stands over him. "You are a coward. A miserable disgusting man." She spits on him, and pulls her dagger from his heaving neck. He tries to speak as his dying body squirms, but no words materialize and he goes still."`

Reviewer 4: ```suggestion
			`	Ildico stands over him. "You are a coward. A miserable disgusting man." She spits on him, and pulls her dagger from his heaving neck. He tries to speak as his dying body squirms, but no words materialize and he goes still.`
```

			`	She turns to you. "I am now a fugitive, and you are my only friend. I need you to get us out of here."`
			`	Ildico walks back to corner far from any of the fighting, and retrieves a bundled baby you'd neglected to notice until now.`
			choice
				`	"You got it. Let's go."`
				`	"Anything for you, Ildy."`
				`	"Sure, but you owe me. Big."`
				`	"No way. You're a murderer. I want nothing to do with you."`
					goto leave
				`	"Stop right there. I am putting you under citizen's arrest until the police arrive."`
					goto arrest

			`	She nods, her face softens, and tears glitter around her eyes. "Thank you, I knew- I knew I could count on you."`
			`	Ildico recollects herself for a moment before pointing to a briefcase lying by a table. "Grab that, and let's go."`
			choice
				`	"Go where?"`

> 			`	She sighs. "Not the Republic, nor the Free Worlds. After tonight I will hunted for this. We need to go to <planet>. And you should probably not let anyone scan your ship until we get there."`

Reviewer 4: ```suggestion
			`	She sighs. "Not the Republic, nor the Free Worlds. After tonight I will be hunted for this. We need to go to <planet>. And you should probably not let anyone scan your ship until we get there."`
```

			`	The two of you escape out into the night and onto your ship.`
				launch

			label arrest
			`	Ildico puts her baby back down, and for just a second you see agony in her eyes. Killing her father or her arranged husband didn't break her mask, but somehow this does. A tear rolls down her cheek. She moves, too fast for you to react, and her dagger is in your throat.`
			`	You lay on the ground, bleeding out, dying. Ildico standing above you, a broken expression on her face. "I thought I had one friend. What a fool I was. I'll never make that mistake again."`

> 			`	She leaves you to die. The world grows dim, and hazy. You remember... you remember... someone....`

Reviewer 4: ```suggestion
			`	She leaves you to die. The world grows dim, and hazy. You remember... you remember... someone...`
```

				`	"Thank you."`
					goto "gotta go"
				`	"Is it wise for you and your child to be left alone in a place like Haven?"`

			`	She laughs. "After surviving on Hope and Chiron? Haven will be a cakewalk."`
				goto end

			label "gotta go"
			`	"No, it is my appreciation that has to be given. You have done what few would have the conviction to do."`

			label end
			`	She takes her child in one hand, and the briefcase in the other, and walks off your ship.`

event "pirateildy"

mission "Ice Queen 7"
	landing
	name "Meet Ildy on <planet>"
	description "See what Ildico needs on <destination>."
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	destination "Haven"
	to offer
		has "event: pirateildy"
	on offer
		dialog
			`Your inbox blinks with a new message from Ildico from Haven:`
			``
			`	"It's been too long, and I could use your help. Please, come to <planet>.`
			`				-your friend`
			`					Ildy"`

	on complete
		dialog
			`Even from orbit, Haven looks ugly, and as you approach the planet it doesn't get any better. Upon landing you're alerted Ildico is waiting for you in the spaceport.`

mission "Ice Queen 8"
	name "Defeat Ildico's rivals"

> 	description "Ildico has requested you to destroy a fleet of marauder ships near the <system> system. Once you're done, return to <planet>."

Reviewer 4: ```suggestion
	description "Ildico has requested you to destroy a fleet of Marauder ships near the <system> system. Once you're done, return to <planet>."
```

			choice
				`	(Duck.)`

			`	You duck, startled by the attack, only to realize she wasn't aiming at you. The rotten-toothed man lies dead on the ground, a bullet hole in his head.`
			`	She looks to the rest of her crew, "Only my friends call me Ildy. And I have only one friend."`
			`	"Yes, boss."`
			`	"Got it, boss."`
			`	"Sure thing, Cap."`
			`	"Whatever you say, boss."`
			`	The group of pirates melt away into the warehouse, one of them dragging the dead body, leaving you and Ildy relatively alone.`
			`	"I'm so glad you came, <first>." Ildico puts a friendly hand on your shoulder and squeezes.`
			`	She turns and waves toward the warehouse. "This is my temporary headquarters. I have few other safe-houses, and I own part of a hangar elsewhere. Not too many ships yet, but I intend to change that soon."`
			`	Ildico holsters her gun right next to where she keeps her dagger and continues. "But making moves is hard. Everyone wants to be on top. Which is why I need you."`
			`	She looks you in the eye and says, "There's a rival gang right next door to us. They have a lot more firepower, but if they could be removed, we could easily double our territory.`

> 			`	"I don't have the ships," she glances at her gang in the distance, "or the quality of crew, to deal with them."`

Reviewer 4: ```suggestion
			`	"I don't have the ships," she glances at her gang in the distance, "or the quality of crew, to deal with them.`
```

			`	"But, I think you do. I can't afford to pay you right now, things are pretty fluid, but I'll owe you. What do you say?"`
			choice
				`	"Sure, I'll help."`
				`	"Fine. But you're going to owe big when this is all over."`
				`	"Anything for you, Ildy."`
				`	"Not interested."`
					goto uninterested

> 			`	Her smile nearly lights up the dim room. "I always know I can count on you. And only you."`

Reviewer 4: ```suggestion
			`	Her smile nearly lights up the dim room. "I always know I can count on you. And only you.`
```

			`	"I fed the gang some false information. They have their entire fleet right now looking for some juicy merchant ships that don't actually exist, so they're not expecting a captain like you to take them on and blow them out of the sky.`
			`	"Check out the nearby systems and look for a ship called the <npc>, which is their flagship. Kill them all, and come back here. To me."`
				accept

			label uninterested
			`	Ildico frowns. She looks down at her feet for a moment, and when she looks back up her face is a mask of rage and hurt. "Then get out."`
				decline

			label why?
			`	Acting defiantly in the face of so much danger, you reach for your sidearm. Unfortunately, it's in vain, because as soon as you feel the pistol grip in your hand you simultaneously feel the burning sensations of a flurry of bullets entering your body and tearing apart your insides.`
				die

	npc kill
		government "Bounty"

> 		personality heroic staying nemesis target

Reviewer 4: ```suggestion
		personality heroic nemesis staying target
```

		system
			distance 1 2
		fleet "Marauder fleet IV"
		dialog phrase "generic hunted bounty fleet eliminated dialog"

	on visit
		dialog phrase "generic fleet bounty hunting on visit"

	on complete
		conversation
			`There's been some intense fighting around the district where Ildico's warehouse was located. A few gunshots echo as you make your way through the rubble and around the fire.`

> 			`	A group of injured, but high-spirited pirates are set up outside the building, and there are bodies littering the nearby streets. An unharmed but bloody Ilidco walks up to you, smiling. She is trailed by a small incongruous toddler with short black hair and lopsided grin.`

Reviewer 4: ```suggestion
			`	A group of injured but high-spirited pirates are set up outside the building, and there are bodies littering the nearby streets. An unharmed but bloody Ildico walks up to you, smiling. She is trailed by a small incongruous toddler with short black hair and lopsided grin.`
```

				goto onlyyou

			label hirand
			`	The little girl lifts a chubby fist up to you.`
			choice
				`	(Fist bump her.)`
					goto bump
				`	(Shake her fist.)`
			`	You awkwardly take her fist in your hand and shake it up and down. Randalin giggles, and babbles something incoherent.`
				goto words
			
			label bump
			`	Her small fist bumps against yours and then she gives you an easy grin and babbles something incoherent.`

			label words
			`	"She's not quite got the hang of words yet. But they'll come," Ildico says approvingly.`
			
			label onlyyou
			`	Ilidco puts a hand on Randalin's shoulder and draws her close. She turns her attention back to the battlefield.`
			`	"This is only the beginning, <first>." Her eyes blaze as she says, "House Drifa will rise again. You'll see. You'll be there."`
			`	She glances back toward her mulling forces. "Thanks again for this, it means a lot that I can count on you, but I have to work to do. Until next time."`
			`	Ildico returns to her group, barking orders and gesturing at the rubble and leaving you to your own devices.`

> mission "News Queen"

Reviewer 4: ```suggestion
mission "Ice Queen: News"
```

	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
		near "Arneb" 10
	to offer
		has "Ice Queen 8: done"
		random < 3
	on offer
		conversation
			`A news kiosk is doing a segment on the growing threat of pirates around Haven. A certain name catches your eye, and you read:`
			`	Self-styled "pirate queen" Ildico Drifa continues to expand her territory and reach. While there is some concern this growing centralization of power could result in greater threats to trade and stability in the Far North, most analysts think it is quite unlikely.`

> 			`	Professor Grenwhich Millstone, foremost authority on the Economy and History of the Northern Pariah worlds, had this to say about Drifa:`

Reviewer 4: Should this be Grenwich?

Author: Probably... =/

Reviewer 4: ```suggestion
			`	Professor Grenwich Millstone, foremost authority on the Economy and History of the Northern Pariah worlds, had this to say about Drifa:`
```

			`	"It is patently inconceivable that any single magnate could consolidate power on Haven. It is culturally anathema, and economically impractical. Frankly, all this talk of a so-called 'pirate queen' is just sensationalist journalism trying to sell the news. In a few weeks another pirate captain will be the new boogieman, and then another."`
			`	The segment ends, and the news kiosk automatically scrolls down to the next article.`
				decline

mission "Skadenga: Blood of Surtr"
	landing
	source "Windblain"
	to offer
		has "Ice Queen 6: done"
		has "outfit: Blood of Surtr"
	on offer
		conversation
			`As your ship descends through Windblain's cloudy atmosphere, you consider landing near Hrithfjall and asking the Ondurdis about the Blood of Surtr.`
			choice
				`	(Take the Blood of Surtr to Hrithfjall.)`
					goto hrithland
				`	(Ask about the Blood of Surtr another time.)`
				`	(Don't visit Hrithfjall at all.)`
					decline

> 			`	You fly past Hjrithfjall without landing and continue on your way.`

Reviewer 4: ```suggestion
			`	You fly past Hrithfjall without landing and continue on your way.`
```

				defer

			label hrithland
			`	You pass numerous new homes built of driftwood and sea-stone as you approach the old village center of Hrithfjall. There is laughter and song in the air and children running everywhere; it always seems larger every time you visit it.`
			`	At the old village hall you find the Ondurdis and Aldfrith deep in conversation with a band of people who stick out beside the Skadenga and Ranfolk; they appear to be foreigners. The Ondurdis's old eyes light up and she gives you a grandmotherly smile as you approach.`
			`	"<first>!" the Ondurdis exclaims happily. "My heart sings to see you."`
			choice
				`	"Who are they?"`
					goto who
				`	"I've come to show you something."`
					goto something
				`	"What do you know about other stones like the Tear of Skade?"`
			`	"On Skade? Only Tears of Skade. But on other worlds? Maybe.`
				goto something

			label who
			`	The Ondurdis steps away from the group and takes you aside, leaving Aldfrith to continue talking with them.`
			`	"They are from Alfheim. They say their culture and world dying and they look for new home before old one gone." She pats your arm gently.`

> 			`	"Word spreads of Hrithfjell and in some corners of Deep old ways still remembered. So they come." Her eyes gleam, "We are like fire on glacier. All see, all come. For warmth, for happiness, for life."`

Reviewer 4: ```suggestion
			`	"Word spreads of Hrithfjell and in some corners of Deep old ways still remembered. So they come." Her eyes gleam, "We are like fire on glacier. All see, all come. For warmth, for happiness, for life.`
```

			label end
			`	The shadows turn a darker shade of purple as you leave the graveyard of the forgotten dead for the familiar confines of your ship.`
				decline

event "stone waiting most"

mission "Stone to Hope"
	name "Bring Tosti's stone to <planet>"
	description "Take Tosti's family stone to <destination>, where the Snaiwaz clan used to live before they immigrated to <origin>."
	to offer
		random < 60
		has "Stone of our Fathers 5: done"
		has "event: stone waiting most"
		not "Ice Queen: offered"
	source "Norn"
	destination "Hope"
	on offer
		conversation

> 			`There is a certain comfort to be found in the familiar swaying of Norn's spaceport. The creaking bridges no longer feel intimidating or threatening, only welcoming. There's never many staff here, but the few you do run into greet you warmly. Word has apparently spread that you've all but been adopted into the Hlewagasti family.`

Author: Not a typo!

			`	Sensing his discomfort, Helm puts an arm on the young man's shoulder. "It's alright, Tosti, we're all friends on Norn."`
			`	The young man staightens up a bit at Helm's reassurance and looks you in the eyes. "My da, well, he liked the waves well enough, but never really settled here, you see? In his heart that is.`
			`	"He never really left our old home. We were refugees and were ever so glad to be taken in by Norn, but he never felt so tall as when he thought about Hope, about home."`
			choice
				`	"What are you going on about?"`
					goto finish
				`	(Wait for him to get the point.)`
					goto finish
				`	"Leave me alone. I'm not interested."`
			`	Crestfallen, the young man stumbles over his words attempting to apologize for the intrusion and slinks away from you in an ungraceful hurry.`
			`	Helm watches him go with an unhappy frown upon his face. "You could have at least heard him out."`
			`	He shakes his head and turns from you before saying, "I thought better of you."`
				decline

			label finish

> 			`	Tosti suddenly stops, and takes a deep breath. His face is red, and beads of sweat glisten on his brow. "We're from Hope, ser. My family. We're refugees and we had to leave everything behind. But da, he never really left Hope."`

Author: He's just a lad speakin' about troubled things from the heart. Conversing and the like aren't his strong suit.

> 			`	The young refugee pulls out a stone criss-crossed in tiny chiseled letters. The familiar engraved story of a life now ended. "This is his stone, and well, he - he'd want his stone left on Hope. I was hoping that was something you could do."`

Author: This is answered later on by Ildico. Parts of Hope were settled by folks from the Deep.

> 			`	The young refugee pulls out a stone criss-crossed in tiny chiseled letters. The familiar engraved story of a life now ended. "This is his stone, and well, he - he'd want his stone left on Hope. I was hoping that was something you could do."`

Author: Oh, maybe you were trying to tell me something. 
 I'll add this as a question you can ask Helm/Tosti and let them answer it for you. I can't right now, but I'll get to it when I get home from work later today.

> 			`	The young refugee pulls out a stone criss-crossed in tiny chiseled letters. The familiar engraved story of a life now ended. "This is his stone, and well, he - he'd want his stone left on Hope. I was hoping that was something you could do."`

Author: Done! Added that question with an answer.

			`	With the words out his eyes drop to the floor and he tenses up as though expecting a blow.`
			choice
				`	"I can take the stone to Hope."`
					goto hopeyes
				`	"I can take the stone to Hope, for a price."`
					goto hopeprice
				`	"Hire a courier. I have more important things to do."`
				`	"No."`

			label hopeno
			`	Helm sighs and tries to pat the young man on the back before he entirely deflates in shame. "Captain <last> is a very busy man. I am sure we can find someone else to make the journey. There is no shame in asking for help."`
			`	Tosti is red and shaking, and mumbles some apologies before slinking away from you and Helm.`
			`	Helm gives you a nod and says, "Thank you for listening, at least."`
				decline

			label hopeprice

> 			`	He grimaces, as if he was just punched in the gut. "I am sorry, ser. We're just poor laborers. Work is hard to come by for us refugees. I thought - I thought I'd just ask. In case you were near. In case it was within your power is all."`

Author: He's asking a stranger for a hand-out. Not only that, but his request is one that the greater human cultural, and even prevailing Deep cultural, would find ridiculous. He's asking you to fly your ship half-way across human space, to an abandoned world, to find a pile of rocks, and drop a rock on top of it. For free.

There aren't loads of captains willing to do that.

> 			`	He grimaces, as if he was just punched in the gut. "I am sorry, ser. We're just poor laborers. Work is hard to come by for us refugees. I thought - I thought I'd just ask. In case you were near. In case it was within your power is all."`

Author: Alright! Added that question and an answer.

	choice
		`	"Who are you?"`
		`	"Are you okay?"`
		`	"What are you doing here?"`

	`	The woman has short jet black hair. She's wearing a military grade, all-purpose survival suit, the kind you might see on deeply embedded special forces. The only weapon on her is a decorated knife at her belt. She looks pale, and frail, probably malnourished. But her icy eyes are clear, and she locks them onto you.`
	`	"You come into my home, and expect me to answer your questions?" Her voice is steady.`
	`	Slowly, awkwardly, she pulls herself to her feet, keeping one hand on the counter for balance. She rests her other hand over her knife.`
	`	"You are lucky that you did not barge in here a year ago, or even a month ago." Despite her bravado there is a deep wariness behind her words. "Now tell me, who are you, and what are you doing here?" she asks, through gritted teeth.`
	choice
		`	(Tell her who you are, and how you were just delivering a stone.)`
			to display
				has "Stone to Hope: done"

> 		`	(Tell her who you are, and how you were just out exploring.)`

Author: Would it make more sense if I just made Stone of Hope of a prerequisite to doing this quest?

> 		`	(Tell her who you are, and how you were just out exploring.)`

Reviewer 5: If you'd rather do that, then sure.

> 		`	(Tell her who you are, and how you were just out exploring.)`

Author: Okay, never mind, lol. I looked at it and this was something Bene cobbled together with branches; of which I don't understand and find extremely intimidating (originally she was found by a dog who doesn't exist anymore so the mission start had to be re-written from scratch). 
Anyways, I've reduced the chances of this mission firing for someone who hasn't done the stone stuff, and added another option as requested.

			`	Ildico seems to deflate. "Ask him? Is he here?"`
			`	The terraformer shakes his head. "Oh, no. He doesn't work for the Academy. But after the rejection he went to the heart of the Republic, I assume to find out why. He now works for them. I see his name from time to time. Just head to the the Republic's main office for the Department of Migration which is located on Chiron."`
			`	Without saying another word, Ildico turns and leaves. You thank Ian for his help and follow her out.`
				accept

mission "Ice Queen 4"
	landing
	invisible
	source "Chiron"
	passengers 1
	to offer
		has "Ice Queen 3: done"
	on offer
		event "deoricdeeds" 1103
		conversation
			`It's easy to imagine Earth once looked and felt much like Chiron does today. A vibrant world, full of engaged and motivated people, all in a hurry to change the universe; or at least, to run it.`
			`	The Department of Migration, Stellar Populations and Advanced Interplanetary Economics is easy enough to find. What's surprising is how quickly you are ushered into a waiting room outside the office of one Mr. Drifa, Sub-Executive of Multiplanetary Pricing.`
			choice
				`	(Wait for Mr. Drifa.)`

> 

Author: It's not just breaking up text; it's breaking up scenes. But, alright.

mission "Ice Queen 6"
	name "Take Ildico to <planet>"
	description "Smuggle Ildico and her child to <destination>."
	source "Chiron"
	destination "Haven"
	passengers 2
	illegal 1250000 `Ildico Drifa is dragged away in chains for high crimes against the Republic and its most honorable servants. Her infant child is taken into the custody of the Republic. You can only watch in sad silence as they are removed from your ship. You wind up quite a bit poorer through a combination of fines and bribes (the breakdown remains somewhat unclear).`
	stealth
	to offer
		has "Ice Queen 5: done"

	on offer
		conversation
			`The directions Ildico gave you take you beyond the bustling city and out into the the large estates owned by the Republic political elite; Drifa Manor sits atop a low hill with what must be a very expensive view of New Sydney. The architecture of the building is reminiscent of elite villas you've seen in the Deep.`
			`	As you approach the main gate you find it wide open, and the attached security booth is abandoned. You could just walk right in.`
			choice

> 				`	(Walk right in.)`

Author: Done!

			`	Ildico stands over him. "You are a coward. A miserable disgusting man." She spits on him, and pulls her dagger from his heaving neck. He tries to speak as his dying body squirms, but no words materialize and he goes still.`
			`	She turns to you. "I am now a fugitive, and you are my only friend. I need you to get us out of here."`
			`	Ildico walks back to corner far from any of the fighting, and retrieves a bundled baby you'd neglected to notice until now.`
			choice
				`	"You got it. Let's go."`
				`	"Anything for you, Ildy."`
				`	"Sure, but you owe me. Big."`
				`	"No way. You're a murderer. I want nothing to do with you."`
					goto leave
				`	"Stop right there. I am putting you under citizen's arrest until the police arrive."`
					goto arrest

			`	She nods, her face softens, and tears glitter around her eyes. "Thank you, I knew- I knew I could count on you."`
			`	Ildico recollects herself for a moment before pointing to a briefcase lying by a table. "Grab that, and let's go."`
			choice

> 				`	"Go where?"`

Reviewer 5: Add another dialog choice here, even if it just leads to the same line.

Author: Okay, done.

Author: Double doned.

			`	She sighs. "Not the Republic, nor the Free Worlds. After tonight I will be hunted for this. We need to go to <planet>. And you should probably not let anyone scan your ship until we get there."`
			`	The two of you escape out into the night and onto your ship.`
				launch

			label arrest
			`	Ildico puts her baby back down, and for just a second you see agony in her eyes. Killing her father or her arranged husband didn't break her mask, but somehow this does. A tear rolls down her cheek. She moves, too fast for you to react, and her dagger is in your throat.`
			`	You lay on the ground, bleeding out, dying. Ildico standing above you, a broken expression on her face. "I thought I had one friend. What a fool I was. I'll never make that mistake again."`

> 			`	She leaves you to die. The world grows dim, and hazy. You remember... you remember... someone...`

Reviewer 2: > You remember... you remember... someone...

This seems unnecessarily vague, what is this supposed to convey?

Author: Neurons exploding in your brain as you die.

			`	She opens the briefcase she took off Chiron, revealing a storage device and an oranate wooden box.`
			`	Ildico gives you a sly smile. "It took years, but the majority of the Drifa fortune has been smuggled into this storage device. They thought I was meek, malleable, and pitiful. Not a threat. They were fools."`
			`	She holds up the box. "And in this, well, this may take some explaining. Before we migrated to Hope, the Drifa were old nobility in the Deep. Legend says we ruled Valhalla before the first ships came. Whatever the truth, my ancestors had this in their possession as a sign of their royalty, this bauble."`
			`	Ildico opens the rune-carved lid of the box, revealing a faintly glowing red rock. "It is called the Blood of Surtr, and it was said to have been the symbol of our rulership, and more importantly, a source of power."`
			`	She shrugs. "I know it sounds far-fetched. But I had this with me on Hope, and I don't think I would have survived without it. When the generators went down, this kept the lights on.`

> 			`	"But, it's time with me is over. I want you, the last friend of House Drifa, and the only person who has shown me any kindness in many years, to have it.`

Reviewer 5: ```suggestion
			`	"But its time with me is over. I want you, the last friend of House Drifa, and the only person who has shown me any kindness in many years, to have it.`
```

			`	"Looks like someone's lost," a voice rises from the group, though you can't tell who's speaking.`
			`	"They look right fancy. Good thing we found them."`
			`	"Whaddya say, Frank, kill 'em now? Or keep 'em for a bit?"`
			choice
				`	"Ildy?"`
				`	"Ildy asked me to come here."`
				`	"Ildy, help!"`
				`	(Grab your sidearm and shoot your way out.)`
					goto why?

			`	The band of cut-throats looks at you, confused, and then most of them lower their weapons.`
			`	"Boss," one of them hollers, "this stranger says they're here for you, I think."`
			`	Footsteps echo on the opposite side of the warehouse as someone approaches.`
			`	One of the men starts laughing. "Ildy?! Haha, never thought to call her that. Hey Ildy! How come you never told us to call you that? Or that some kind of special pet name?"`
			`	The man flashes an unsavory grin at you, with his rotting teeth on full display.`
			`	In the dim light the footsteps materialize into a woman in her mid-thirties with hair darker than vacuum. Ildico Drifa gives you a warm smile, then raises up her hand and fires her side-arm.`
			choice

> 				`	(Duck.)`

Reviewer 5: Again, try not to have single choices if possible.

Author: I disagree with the blanket reasoning for this approach (I know it's a style guide thing). In nearly all video games, even ones where you appear to have endless choices, there are times where you really only one have one. You can only go through one door, or you can only walk in one direction, or you can only have one interaction at a crucial moment. The better games do not automate these moments for you, but force you to make the only choice you're allowed to make. To do it for you is to make you passive, is to regrade the medium for interactive to acted upon.

That said, I agree more options are better here, and have done as requested.

			`	She looks to the rest of her crew, "Only my friends call me Ildy. And I have only one friend."`
			`	"Yes, boss."`
			`	"Got it, boss."`
			`	"Sure thing, Cap."`
			`	"Whatever you say, boss."`
			`	The group of pirates melt away into the warehouse, one of them dragging the dead body, leaving you and Ildy relatively alone.`
			`	"I'm so glad you came, <first>." Ildico puts a friendly hand on your shoulder and squeezes.`
			`	She turns and waves toward the warehouse. "This is my temporary headquarters. I have few other safe-houses, and I own part of a hangar elsewhere. Not too many ships yet, but I intend to change that soon."`
			`	Ildico holsters her gun right next to where she keeps her dagger and continues. "But making moves is hard. Everyone wants to be on top. Which is why I need you."`
			`	She looks you in the eye and says, "There's a rival gang right next door to us. They have a lot more firepower, but if they could be removed, we could easily double our territory.`
			`	"I don't have the ships," she glances at her gang in the distance, "or the quality of crew, to deal with them.`

> 			`	"But, I think you do. I can't afford to pay you right now, things are pretty fluid, but I'll owe you. What do you say?"`

Reviewer 5: ```suggestion
			`	"But I think you do. I can't afford to pay you right now, things are pretty fluid, but I'll owe you. What do you say?"`
```

			`	"Perfect. They never had a chance." She wipes the blood off her dagger, and notices your gaze.`
			`	"The safest place for her was by my side," she says matter-of-factly to you, then she kneels down to her daughter. "Randalin, this is <first>. You met them long ago when you were a baby. <first> is my friend."`
			`	She stands up again and looks back at you. "And <first>, this is Randalin. She's grown a bit since you saw her last."`
			choice
				`	"Hi, Randalin. It's wonderful to meet you."`
					goto hirand
				`	(Nod at the child.)`
					goto onlyyou
				`	"She was with you for the battle?"`
			`	Ildico shrugs. "Who would I trust with her? Who could I trust with her? Only you, <first>, and you were elsewhere."`
				goto onlyyou

			label hirand
			`	The little girl lifts a chubby fist up to you.`
			choice
				`	(Fist bump her.)`
					goto bump
				`	(Shake her fist.)`
			`	You awkwardly take her fist in your hand and shake it up and down. Randalin giggles, and babbles something incoherent.`
				goto words
			
			label bump

> 			`	Her small fist bumps against yours and then she gives you an easy grin and babbles something incoherent.`

Reviewer 5: ```suggestion
			`	Her small fist bumps against yours. She then gives you an easy grin and babbles something incoherent.`
```

			label words
			`	"She's not quite got the hang of words yet. But they'll come," Ildico says approvingly.`
			
			label onlyyou
			`	Ilidco puts a hand on Randalin's shoulder and draws her close. She turns her attention back to the battlefield.`
			`	"This is only the beginning, <first>." Her eyes blaze as she says, "House Drifa will rise again. You'll see. You'll be there."`
			`	She glances back toward her mulling forces. "Thanks again for this, it means a lot that I can count on you, but I have to work to do. Until next time."`

> 			`	Ildico returns to her group, barking orders and gesturing at the rubble and leaving you to your own devices.`

Reviewer 5: ```suggestion
			`	Ildico returns to her group, barking orders and gesturing at the rubble, leaving you to your own devices.`
```

mission "Ice Queen: News"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
		near "Arneb" 10
	to offer
		has "Ice Queen 8: done"
		random < 3
	on offer
		conversation
			`A news kiosk is doing a segment on the growing threat of pirates around Haven. A certain name catches your eye, and you read:`
			`	Self-styled "pirate queen" Ildico Drifa continues to expand her territory and reach. While there is some concern this growing centralization of power could result in greater threats to trade and stability in the Far North, most analysts think it is quite unlikely.`

> 			`	Professor Grenwich Millstone, foremost authority on the Economy and History of the Northern Pariah worlds, had this to say about Drifa:`

Reviewer 5: ```suggestion
			`	Professor Grenwich Millstone, foremost authority on the economy and history of the northern pirate worlds, had this to say about Drifa:`
```

> 			`	"It is patently inconceivable that any single magnate could consolidate power on Haven. It is culturally anathema, and economically impractical. Frankly, all this talk of a so-called 'pirate queen' is just sensationalist journalism trying to sell the news. In a few weeks another pirate captain will be the new boogieman, and then another."`

Reviewer 5: ```suggestion
			`	"It is patently inconceivable that any single magnate could consolidate power on Haven. It is culturally anathema, and economically impractical. Frankly, all this talk of a so-called 'pirate queen' is just sensationalist journalism trying to sell the news. In a few weeks another pirate captain will be the new bogeyman, and then another."`
```

			`	The segment ends, and the news kiosk automatically scrolls down to the next article.`
				decline

mission "Skadenga: Blood of Surtr"
	landing
	source "Windblain"
	to offer
		has "Ice Queen 6: done"
		has "outfit: Blood of Surtr"
	on offer
		conversation

> 			`As your ship descends through Windblain's cloudy atmosphere, you consider landing near Hrithfjall and asking the Ondurdis about the Blood of Surtr.`

Reviewer 5: Correct me if I'm wrong, but it seems that you don't actually need to learn about the Skadenga or Ondurdis to get this mission.

Author: You're right.

Author: Thinking about this. I could just make it so that doing the Skadegna missions is a prerequisite for these.

			choice
				`	(Take the Blood of Surtr to Hrithfjall.)`
					goto hrithland
				`	(Ask about the Blood of Surtr another time.)`
				`	(Don't visit Hrithfjall at all.)`
					decline

			`	You fly past Hrithfjall without landing and continue on your way.`
				defer

			label hrithland
			`	You pass numerous new homes built of driftwood and sea-stone as you approach the old village center of Hrithfjall. There is laughter and song in the air and children running everywhere; it always seems larger every time you visit it.`

> 			`	At the old village hall you find the Ondurdis and Aldfrith deep in conversation with a band of people who stick out beside the Skadenga and Ranfolk; they appear to be foreigners. The Ondurdis's old eyes light up and she gives you a grandmotherly smile as you approach.`

Reviewer 5: Who are the Ranfolk?

Author: The people of Ran you meet when you migrate the Skadenga to Windblain.

Author: I've added 'indigenous' in front of Ranfolk to help remind the player.

Reviewer 5: My bad.

			choice
				`	"What happened to them?"`
				`	"Where could I find them?"`

			`	Aldfrith shakes his head slowly. "In the time of my grandfather the men from the Deep came and took them off world. They said they were too valuable to belong in the hands of savages, and since then we've never found another Scale of Ran. I think she was displeased we allowed them to be taken."`
			`	The old man pauses for a moment and then says, "Although... I do remember when I was a child the folk from the Deep coming and visiting us again. They had questions about... our baubles. I think they said they did not work? They thought they were just useless charms. Perhaps the Scales stopped working?"`
			`	He shrugs. "I am sorry, <first>, that I cannot tell you more."`
			`	The Ondurdis puts a hand on his back. "I value your stories." She turns her attention back to you, "Maybe all God-Stones once produced wondrous stones of power."`
			`	She points at the Blood of Surtr. "Maybe that is where yours came from. But more than guesses, we cannot provide."`

> 			choice
> 				`	(Take your leave.)`
> 

Reviewer 5: ```suggestion
```

				decline

mission "Stone to Hope: Thanks"
	source "Norn"
	to offer
		has "Stone to Ice Queen: offered"
	on offer
		conversation
			`In the quiet, creaking spaceport, it's easy enough to spot Helm doing some much needed maintenance on the ships that have landed here for the night. The last time you met him, he asked you to bring a stone to Hope. Do you want to check in with him?`
			choice
				`	(Go over to Helm and tell about the stone.)`
					goto tell
				`	(Leave Helm alone.)`
					decline
			
			label tell
			`	As you say your hello you accidentally startle Helm, who's in the middle of recalibrating a repulsor.`
			`	"Woah! <first>! No need to sneak up on me!" He stands up to greet you. "What's new?"`
			choice
				`	"I found the Family-Cairn and left the stone there."`
					goto where
					to display
						has "placed tosti's stone"
				`	"I chucked the stone out of a hatch, but it's at least on Hope now."`
					goto threw
					to display
						has "chucked tosti's stone"
				`	"I threw the stone off a cliff, but it's at least on Hope now."`
					goto threw
					to display
						has "threw tosti's stone"
				`	"Where's Tosti?"`
			label where

> 			`	"Ah, Tosti's out on the seas. The lad got himself a job working on a fishing trawler. It'd likely be a few years before he'll call to port here again."`

Reviewer 5: This first sentence feels a bit incongruous with the first dialog choice.

Author: I have made some edits to address this!

			`	Ildico stands over him. "You are a coward. A miserable disgusting man." She spits on him, and pulls her dagger from his heaving neck. He tries to speak as his dying body squirms, but no words materialize and he goes still.`
			`	She turns to you. "I am now a fugitive, and you are my only friend. I need you to get us out of here."`
			`	Ildico walks back to corner far from any of the fighting, and retrieves a bundled baby you'd neglected to notice until now.`
			choice
				`	"You got it. Let's go."`
				`	"Anything for you, Ildy."`
				`	"Sure, but you owe me. Big."`
				`	"No way. You're a murderer. I want nothing to do with you."`
					goto leave
				`	"Stop right there. I am putting you under citizen's arrest until the police arrive."`
					goto arrest

			`	She nods, her face softens, and tears glitter around her eyes. "Thank you, I knew- I knew I could count on you."`
			`	Ildico recollects herself for a moment before pointing to a briefcase lying by a table. "Grab that, and let's go."`
			choice
				`	"Go where?"`
					goto where
				`	"What's in the briefcase?"`

> 			`	The recently widowed women gives you a suspecious look, but then nods, "What I need to survive." She pauses, "Now grab it, and let's go."`

Reviewer 4: Why on separate paragraphs?

> 			`	The recently widowed women gives you a suspecious look, but then nods, "What I need to survive." She pauses, "Now grab it, and let's go."`

Reviewer 3: 'She pauses and says' felt slightly clunky, you can have the same effect with a new paragraph.

It is my general feeling (not sure if it is a rule) that only verbs representing speech precede a comma when dialogue occurs. Therefore, 'She pauses, "Now grab it, and let's go."' feels wrong, while 'She pauses and says, "Now grab it, and let's go."' is correct yet ehhhh

mission "Ice Queen 6"
	name "Take Ildico to <planet>"
	description "Smuggle Ildico and her child to <destination>."
	source "Chiron"
	destination "Haven"
	passengers 2
	illegal 1250000 `Ildico Drifa is dragged away in chains for high crimes against the Republic and its most honorable servants. Her infant child is taken into the custody of the Republic. You can only watch in sad silence as they are removed from your ship. You wind up quite a bit poorer through a combination of fines and bribes (the breakdown remains somewhat unclear).`
	stealth
	to offer
		has "Ice Queen 5: done"

	on offer
		conversation
			`The directions Ildico gave you take you beyond the bustling city and out into the the large estates owned by the Republic political elite; Drifa Manor sits atop a low hill with what must be a very expensive view of New Sydney. The architecture of the building is reminiscent of elite villas you've seen in the Deep.`
			`	As you approach the main gate you find it wide open, and the attached security booth is abandoned. You could just walk right in.`
			choice
				`	(Walk right in.)`
					goto walk
				`	(Investigate before entering.)`

> 			`	The gate itself apppears to have been opened from the inside, not broken into, and there are no signs of a struggle in the security booth. The employee parking beyond is also completely empty. It seems everyone left fast and didn't bother to close the gate behind them. You're entirely alone out here.`

Reviewer 3: ```suggestion
			`	The gate itself appears to have been opened from the inside, not broken into, and there are no signs of a struggle in the security booth. The employee parking beyond is also completely empty. It seems everyone left fast and didn't bother to close the gate behind them. You're entirely alone out here.`
```
forgot this one

					goto hopeprice
				`	"I thought Family-Cairns were exclusive to the Deep. Why would there be one on Hope?"`
					goto whyhope
					to display
						not "stones"
				`	"Hire a courier. I have more important things to do."`
				`	"No."`

			label hopeno
			action
				clear "stones"
			`	Helm sighs and tries to pat the young man on the back before he entirely deflates in shame. "Captain <last> is a very busy man. I am sure we can find someone else to make the journey. There is no shame in asking for help."`
			`	Tosti is red and shaking, and mumbles some apologies before slinking away from you and Helm.`
			`	Helm gives you a nod and says, "Thank you for listening, at least."`
				decline

			label whyhope
			action
				set "stones"
			`	"Lots of the first folks who settled Hope came from the Deep." Helm answers, getting a far off look in his eyes. "It wasn't like most worlds in the south. It was going to be different. At least, that's what they believed."`
			`	He shrugs, "Many of them didn't forget where they came from. Brought the old ways with them. Some refugees, like Tosti's family, even found there way back here when Hope was lost.`

> 			`	"Anyways, that's why you can find Family-Cairns on Hope - because our people were there. Now, getting back to the matter at hand, you willing to help Tosti out?"`

Author: Oops, fixed that,

				`	"You're welcome, Ildico."`

			`	She looks at you for just a moment, as if in thought, and then says, "If we ever meet again, call me Ildy. It's what my friends call me."`
			`	With that you take your leave.`
				decline

event "deoricdeeds"

mission "Ice Queen 5"
	landing
	name "Visit Ildico"
	description "Ildico has asked you to visit her on <destination>."
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	destination "Chiron"
	to offer
		has "event: deoricdeeds"

	on offer
		dialog
			`Your inbox blinks with a new message from Ildico on Chiron:`
			``
			`	"I need to see you. Please come. My address is attached.`
			`				-your friend`
			`					Ildy"`

	on complete
		dialog
			`The myriad lights of urban settlements twinkle in the dusk as you approach the capital, and main spaceport, of Chiron. Crickets, brought from old Earth, chirp in the twilight air. Your ship parked, you'll need to make your way to the main spaceport to hire transportation to Ildico's address.`

mission "Ice Queen 6"
	name "Take Ildico to <planet>"
	description "Smuggle Ildico and her child to <destination>."
	source "Chiron"
	destination "Haven"
	passengers 2

> 	illegal 1250000 `Ildico Drifa is dragged away in chains for high crimes against the Republic. You can only watch in silence as they are removed from your ship and her infant child is taken into custody. You end up fined <fine> for your actions.`

Reviewer 5: Sorry, I don't think <fine> is actually a thing. My bad.
```suggestion
	illegal 1250000 `Ildico Drifa is dragged away in chains for high crimes against the Republic. You can only watch in silence as they are removed from your ship and her infant child is taken into custody. You end up fined for your actions.`
```

Author: would be nice if it was a thing ;)

	stealth
	to offer
		has "Ice Queen 5: done"

	on offer
		conversation

> 			`The directions Ildico gave you take you beyond the bustling city and out into the the large estates owned by the Republic political elite; Drifa Manor sits atop a low hill with what must be a very expensive view of New Sydney. The architecture of the building is reminiscent of elite villas you've seen in the Deep.`

Reviewer 5: ```suggestion
			`The directions Ildico gave you take you beyond the bustling city and out into the the large estates owned by the Republic political elite; Drifa Manor sits atop a low hill with what must be a very expensive view of New Sydney. The architecture of the building is reminiscent of upper-class villas you've seen in the Deep.`
```

			scene "outfit/surtrblood"
			`	"I'll also transfer a small sum of money for your troubles. I fear I will be needing most of my family's fortune, but hopefully what little I can offer will help."`
			choice
				`	"Thank you."`
					goto "gotta go"
				`	"Is it wise for you and your child to be left alone in a place like Haven?"`

			`	She laughs. "After surviving on Hope and Chiron? Haven will be a cakewalk."`
				goto end

			label "gotta go"
			`	"No, it is my appreciation that has to be given. You have done what few would have the conviction to do."`

			label end
			`	She takes her child in one hand, and the briefcase in the other, and walks off your ship.`

event "pirateildy"

mission "Ice Queen 7"
	landing
	name "Meet Ildy on <planet>"
	description "See what Ildico needs on <destination>."
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	destination "Haven"
	to offer
		has "event: pirateildy"
	on offer
		dialog
			`Your inbox blinks with a new message from Ildico from Haven:`
			``
			`	"It's been too long, and I could use your help. Please, come to <planet>.`

> 			`				-your friend`

Reviewer 5: ```suggestion
			`				- Your friend,`
```

			`					Ildy"`

	on complete
		dialog
			`Even from orbit, Haven looks ugly, and as you approach the planet it doesn't get any better. Upon landing you're alerted Ildico is waiting for you in the spaceport.`

mission "Ice Queen 8"
	name "Defeat Ildico's rivals"
	description "Ildico has requested you to destroy a fleet of Marauder ships near the <system> system. Once you're done, return to <planet>."
	source "Haven"
	to offer
		has "Ice Queen 7: done"
	on offer
		conversation

> 			`You arrive at the address Ildico gave you; a large warehouse on the edge of the spaceport. When you step into the dimly lit building, your eyes struggle to adjust, gradually you realize you're face-to-face with a group of dirty burly men and women leveling a half-dozen weapons right at you.`

Reviewer 5: ```suggestion
			`You arrive at the address Ildico gave you; a large warehouse on the edge of the spaceport. When you step into the dimly lit building, your eyes struggle to adjust, and gradually you realize you're face-to-face with a group of dirty burly men and women leveling a half-dozen weapons right at you.`
```

			`	"This is only the beginning, <first>." Her eyes blaze as she speaks, "House Drifa will rise again. You'll see. You'll be there."`
			`	She glances back toward her mulling forces. "Thanks again, it means a lot that I can count on you, but I have to work to do. Until next time."`
			`	Ildico returns to her group, barking orders and gesturing at the rubble, leaving you to your own devices.`

mission "Ice Queen: News"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
		near "Arneb" 10
	to offer
		has "Ice Queen 8: done"
		random < 3
	on offer
		conversation
			`A news kiosk is doing a segment on the growing threat of pirates around Haven. A certain name catches your eye, and you read:`
			`	Self-styled "pirate queen" Ildico Drifa continues to expand her territory and reach. While there is some concern this growing centralization of power could result in greater threats to trade and stability in the Far North, most analysts think it is quite unlikely.`
			`	Professor Grenwich Millstone, foremost authority on the economy and history of the northern pirate worlds, had this to say about Drifa:`

> 			`	"It is patently inconceivable that any single magnate could consolidate power on Haven. It is culturally anathema, and economically impractical. Frankly, all this talk of a so-called 'pirate queen' is just sensationalist journalism trying to sell the news. In a few weeks another pirate captain will be the new bogeyman, and then another."`

Reviewer 5: ```suggestion
			`	"It is patently inconceivable that any single magnate could consolidate power on Haven. It is cultural anathema, and economically impractical. Frankly, all this talk of a so-called 'pirate queen' is just sensationalist journalism trying to sell the news. In a few weeks another pirate captain will be the new bogeyman, and then another."`
```

			`	"<first>!" the Ondurdis exclaims happily. "My heart sings to see you."`
			choice
				`	"Who are they?"`
					goto who
				`	"I've come to show you something."`
					goto something
				`	"What do you know about other stones like the Tear of Skade?"`
					to display
						not "tears"
			`	"On Skade? Only Tears of Skade. But on other worlds? Maybe."`
				goto something

			label who
			`	The Ondurdis steps away from the group and takes you aside, leaving Aldfrith to continue talking with them.`
			`	"They are from Alfheim. They say their culture and world dying and they look for new home before old one gone." She pats your arm gently.`
			`	"Word spreads of Hrithfjell and in some corners of Deep old ways still remembered. So they come." Her eyes gleam, "We are like fire on glacier. All see, all come. For warmth, for happiness, for life.`
			`	"But, <first>, you have not come all this way to talk with old woman. What brings you here?"`

			label something
			action
				clear "tears"
			`	You show her the Blood of Surtr, and tell her the story of House Drifa taking it from the Deep.`

> 			`	The Ondurdis nods slowly. "I only know of Tears of Skade. Our Goddess cried, maybe others did as well?"`

Reviewer 5: ```suggestion
			`	The Ondurdis nods slowly. "I only know of Tears of Skade. Our Goddess cried - maybe others did as well?"`
```

> 			`	She turns and motions to Aldfrith who steps over to you and asks, "Did Ran ever cry cold tears?"`

Reviewer 5: ```suggestion
			`	She turns and motions to Aldfrith who steps over to you. She asks, "Did Ran ever cry cold tears?"`
```

			`	With the words out his eyes drop to the floor and he tenses up as though expecting a blow.`
			label hopeoptions
			choice
				`	"I can take the stone to Hope."`
					goto hopeyes
				`	"I can take the stone to Hope, for a price."`
					goto hopeprice
				`	"I thought Family-Cairns were exclusive to the Deep. Why would there be one on Hope?"`
					goto whyhope
					to display
						not "stones"
				`	"Hire a courier. I have more important things to do."`
				`	"No."`

			label hopeno
			action
				clear "stones"
			`	Helm sighs and tries to pat the young man on the back before he entirely deflates in shame. "Captain <last> is a very busy man. I am sure we can find someone else to make the journey. There is no shame in asking for help."`
			`	Tosti is red and shaking, and mumbles some apologies before slinking away from you and Helm.`
			`	Helm gives you a nod and says, "Thank you for listening, at least."`
				decline

			label whyhope
			action
				set "stones"

> 			`	"Lots of the first folks who settled Hope came from the Deep." Helm answers, getting a far off look in his eyes. "It wasn't like most worlds in the south. It was going to be different. Atleast, that's what they believed."`

Reviewer 4: ```suggestion
			`	"Lots of the first folks who settled Hope came from the Deep." Helm answers, getting a far off look in his eyes. "It wasn't like most worlds in the south. It was going to be different. At least, that's what they believed."`
```

> 			`	He shrugs, "Many of them didn't forget where they came from. Brought the old ways with them. Some refugees, like Tosti's family, even found there way back here when Hope was lost."`

Reviewer 4: ```suggestion
			`	He shrugs, "Many of them didn't forget where they came from. Brought the old ways with them. Some refugees, like Tosti's family, even found there way back here when Hope was lost.`
```

			`	Ian shrugs. "Though he could afford it, there were other concerns."`
			`	Ildico steps forward and looms over Ian like an angry shadow. It appears that it is taking all of her willpower to keep her from ripping this man apart. "Concerns?"`
			`	"Y-yes. I don't know the specifics. You have to understand, here at the Academy it's not just about money. It's about politics, and power. I am just a Second Degree Terraformer, and after the initial meeting I wasn't privy to the details. But someone, or some government or corporation didn't want it. If you want to know more you'll have to ask him yourself."`

> 			`	Ildico seems to deflate. "Ask him? Is...is he here?"`

Reviewer 4: ```suggestion
			`	Ildico seems to deflate. "Ask him? Is... is he here?"`
```

			choice
				`	"Go where?"`

> 				`	"Go? You murdered a high ranking executive in the Republic administration. No where in the Republic will be safe."`

Reviewer 4: ```suggestion
				`	"Go? You murdered a high ranking executive in the Republic administration. Nowhere in the Republic will be safe."`
```

				`	"What's in the briefcase?"`
			`	The recently widowed woman gives you a suspicious look, but then nods. "What I need to survive.`
			`	"Now grab it, and let's go."`
			choice
				`	"Go where?"`
					goto where

> 				`	(Grab the briefcase).`

Reviewer 4: ```suggestion
				`	(Grab the briefcase.)`
```

			`	The old man pauses for a moment and then says, "Although... I do remember when I was a child the folk from the Deep coming and visiting us again. They had questions about... our baubles. I think they said they did not work? They thought they were just useless charms. Perhaps the Scales stopped working?"`
			`	He shrugs. "I am sorry, <first>, that I cannot tell you more."`
			`	The Ondurdis puts a hand on his back. "I value your stories." She turns her attention back to you, "Stone-magic fades when outsiders come, this is known. Maybe all God-Stones once gave such gifts, but no longer."`
			`	She points at the Blood of Surtr. "Surtr, is known, he is volcano that erupts on ice. He is black fire that swallows world. But he had no God-Stone on Nifel." She pauses and looks at Aldfrith.`
			`	"Oh, uh, and none here. Ran, like the sea, is jealous and would share no world with another." Aldfrith says quickly.`

> 			`	"Maybe on other world, there is Surtr God-Stone, and from it he bleeds. More than that, I do not know." She says apologetically.`

Reviewer 4: ```suggestion
			`	"Maybe on other world, there is Surtr God-Stone, and from it he bleeds. More than that, I do not know," she says apologetically.`
```
